### PR TITLE
[EC-383] Create io-sign Storage Account replica in ItalyNorth

### PR DESCRIPTION
**Pull request by Krusty93**
### List of changes

Create replica in ITN with its own resource group.
Replicated all settings, queues and containers.
Enable versioning on both SA
Enable change feed on WEU SA

### Motivation and context

ItalyNorth migration

### Type of changes

- [X] Add new resources
- [X] Update configuration to existing resources
- [ ] Remove existing resources

### Env to apply

- [ ] DEV
- [ ] UAT
- [X] PROD

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [X] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [X] No

### Other information

---

### How to apply

After PR is approved
1. run deploy pipeline from Azure DevOps [io-platform-iac-projects](https://dev.azure.com/pagopaspa/io-platform-iac-projects/_build?view=folders)
2. select PR branch
3. wait for approval

### CODE REVIEW

File: src/domains/sign/env/prod/terraform.tfvars [hunk 1]
```diff
@@ -30,7 +30,8 @@ subnets_cidrs = {
 }
 
 storage_account = {
-  enable_versioning             = false
+  enable_versioning             = true
+  change_feed_enabled           = true
```
Comment (AleDore) [inline src/domains/sign/env/prod/terraform.tfvars:34]: why enabling change feed?

File: src/domains/sign/storage.tf [hunk 2]
```diff
@@ -6,6 +6,7 @@ module "io_sign_storage" {
   account_replication_type        = var.storage_account.replication_type
   access_tier                     = "Hot"
   blob_versioning_enabled         = var.storage_account.enable_versioning
+  blob_change_feed_enabled        = var.storage_account.change_feed_enabled
```
Comment (AleDore) [inline src/domains/sign/storage.tf:9]: ditto

File: src/domains/sign/env/prod/terraform.tfvars (hunk 1 above)
Comment (Krusty93) [reply src/domains/sign/env/prod/terraform.tfvars:34]: It is required to enable object replication

**Review by AleDore — APPROVED** [review]

**Review by christian-calabrese — APPROVED** [review]
LGTM

**Pull request merged**